README.md

```diff
@@ -255,8 +255,8 @@ We provide `buildkitd` container images as [`moby/buildkit`](https://hub.docker.
 To run daemon in a container:
 
 ```
-docker run -d --privileged -p 1234:1234 moby/buildkit:latest --addr tcp://0.0.0.0:1234
-export BUILDKIT_HOST=tcp://0.0.0.0:1234
+docker run -d --privileged --name buildkit moby/buildkit:latest
+export BUILDKIT_HOST=docker://buildkit
```

Comment:
As this points to `0.4.0` it wouldn't actually work on merge. Maybe either delay this until 0.5 or add another example with `moby/buildkit:master`

Reply:
reverted